packages/template-retail-react-app/app/components/otp-auth/index.jsx

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useState, useRef, useEffect} from 'react'
+import PropTypes from 'prop-types'
+import {FormattedMessage} from 'react-intl'
+import {
+    Button,
+    Input,
+    SimpleGrid,
+    Stack,
+    Text,
+    HStack,
+    Modal,
+    ModalBody,
+    ModalCloseButton,
+    ModalContent,
+    ModalHeader,
+    ModalOverlay
+} from '../shared/ui'
```

Comment:
Run `npm run lint:fix` to auto-fix the linting error here. Unique to the template, import paths need to be absolute urls.

```suggestion
} from '@salesforce/retail-react-app/app/components/shared/ui'
```